src/tools/diagnostic-messages.md

```diff
@@ -5644,7 +5644,7 @@ class C {
 }
 {% endprettify %}
 
-If the instances of the class should be mutable, then remove the
+If the instances of the class should be mutable, then remove the `@immutable` annotation:
```

Comment:
Thanks. I dropped a space in the source code from which this was generated which kept a whole line of text from being included in the generated output. The original reviewed text was:

If the instances of the class should be mutable, then remove the annotation, or choose a different superclass if the annotation is inherited:

The original source code will be fixed by https://dart-review.googlesource.com/c/sdk/+/171810. If you could apply the fix manually, that would be much appreciated.

Reply:
Great, I updated it with the original source code comment.